src/auth/models/authOptions.interface.ts

```diff
@@ -0,0 +1,16 @@
+export interface AuthOptions {
+    authorizationURL: string
```

Comment:
Capitalisation consistency... in some places, the names use `somethingURL` and in others it's `somethingUrl` - should it not be one or the other?

Reply:
Done

Reply:
Resolved - @udaydenduluri33 explained that `tokenURL` and `callbackURL` are inherited from a base class in a library outside of our control.